node/src/service.rs

```diff
@@ -1,46 +1,80 @@
-use cumulus_client_consensus_relay_chain::{
```

Comment:
I didn't really review this file. Is there anything in it other than bringing us up to date with latest substrate/cumulus?

Reply:
no, there is nothing else than updating to latest changes.